File: HelloWorld/results/HelloWorld.pl
use strict;
use warnings;

package TestClass;

sub new
{
    my $class = shift;
    my $self = {};
    bless $self, $class;
    return $self;
}

sub test_method {
    my ( $self ) = @_;
    my $value = 1 + 2 * 3 - 4;
    my $map_ = {
      a => 5,
      b => 6,
    };
    my $text = "Hello world! value = @{[$value]}, map[a] = @{[${$map_}{\"a\"}]}";
    print(($text) . "\n");
}

package Program;

eval {
    my $c = new TestClass();
    $c->test_method();
};
if ($@) {
    print "Exception: " . $@
}

File: HelloWorld/results/HelloWorld.cs
using System.Collections.Generic;
using System.Linq;
using System;

public class TestClass
{
    public void TestMethod()
    {
        var value = 1 + 2 * 3 - 4;
        var map_ = new Dictionary<string, int>
        {
            { "a", 5 },
            { "b", 6 }
        };
        var text = $"Hello world! value = {value}, map[a] = {map_["a"]}";
        Console.WriteLine(text);
    }
}

public class Program
{
    static public void Main(string[] args)
    {
        try 
        {
            new TestClass().TestMethod();
        }
        catch (System.Exception e)
        {
            System.Console.WriteLine($"Exception: {e.Message}");
        }
    }
}